MinPlatformPkg/MinPlatformPkg.dsc

```diff
@@ -215,6 +215,7 @@
   MinPlatformPkg/Test/TestPointStubDxe/TestPointStubDxe.inf
   MinPlatformPkg/Test/TestPointDumpApp/TestPointDumpApp.inf
 
+  MinPlatformPkg/Tcg/Library/PeiDxeTpmPlatformHierarchyLib/PeiDxeTpmPlatformHierarchyLib.inf
```

Comment:
Since this depends on TPM 2, would it make sense to move down a line, inside the !if ?

Reply:
I left it outside because that flag is almost never set to TRUE and since this is testing the build it can introduce a silent build break. I think it makes sense in the FDF because you obviously don't want drivers you don't need in the final flash image.